src/data/nat/digits.lean

```diff
@@ -414,3 +418,42 @@ begin
   rw of_digits_neg_one at t,
   exact t,
 end
+
+lemma digits_len_le_digits_len_succ (b n : ℕ) : (digits b n).length ≤ (digits b (n + 1)).length :=
+begin
+  cases b,
+  { -- base 0
+    cases n; simp,
+  },
```

Comment:
```suggestion
    cases n; simp, },
```
Here and below: we generally put closing braces at the end of the previous line.

Reply:
@SnobbyDragon , I think Bryan meant that all of the closing braces in the file, for example L447-450 would be at the end of 446.

so this

```
          refl,
        },
      },
    },
  },
```

becomes

```
          refl, }, }, }, },
```

and I don't think you need the commas there either

Reply:
@shingtaklam1324 That is indeed what I meant - thanks for elaborating!

By the way, I believe this conversation was most likely automatically marked as resolved by GitHub when my suggestion was committed. While it doesn't hurt, it's also not strictly necessary to unresolve such conversations if it's likely that the author can finish off the requests without needing to discuss things further (e.g. trivial style changes, like this one).

Reply:
Oh right, I forgot that GitHub automatically resolves a conversation when the suggestion is merged. GitHub said "SnobbyDragon marked this conversation as resolved" so I thought Angela marked it herself. 🤦

Reply:
Now it looks good, so I'll set this as resolved!

One thing I forgot to point out is that sometimes we leave the last comma before the braces start (you might have noticed that style elsewhere in this file). That way you can see "goals accomplished" if you put their cursor there. We're not consistent about this so it's not a big deal. I definitely wouldn't bother changing that here unless you personally prefer it.